### ISSUE TITLE
HTTPS : Let's encrypt !

### ISSUE THREAD

**Issue by drdeteck** [poma/hotsapi#28]
Hey guys, me again, it would be cool if you could encrypt the host with Let's encrypt !

https://certbot.eff.org/

Tks, keep up the good work !  ❤️

**Comment by ekzor** [comment]
I'm a strong advocate for web privacy, encryption, and use of HTTPS; I like this idea.

However, all the data in the API is available for public use so it's unclear to me what the encryption will be protecting in this context. I'm also concerned that this may lead to unnecessary overhead resource consumption, although my knowledge is a bit outdated in this realm and perhaps encryption isn't as CPU-heavy as it used to be.

To summarize: I like encryption, but does it make sense for this application?

**Comment by snowczar** [comment]
I agree with ekzor - because we are not sending private information I don't see encryption as being a high priority at this time.

**Comment by poma** [comment]
Closing until someone convinces me that we need HTTPS

**Comment by poma** [comment]
Found free SSL certs from GlobalSign, now hotsapi supports https

**Comment by tbuehlmann** [comment]
There's no good reason not to support HTTPS. Good call on supporting this earlier than later.
